deploy/helm/sumologic/templates/clusterrolebinding.yaml

```diff
@@ -2,12 +2,13 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: {{ template "sumologic.metadata.name.roles.clusterrolebinding" . }}
+  namespace: {{ template "sumologic.namespace"  . }}
```

Comment:
`ClusterRoleBinding`s are not namespaced
```suggestion
```

Reply:
@astencel-sumo This has been now removed. Earlier goldentest needed it so we added it here but ideally yes we don't need namsepace for non-namespaced resource.